jobs/pr_gate/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright 2016, EMC, Inc.
+
+# set -x
+
+rm -f /var/lib/dhcp/dhchp.leases
+touch /var/lib/dhcp/dhcpd.leases
+chown root:root /var/lib/dhcp/dhcpd.leases
+chmod 666 /var/lib/dhcp/dhcpd.leases
+
+service isc-dhcp-server stop
+
+mongod &
+sleep 1
+rabbitmq-server &
+sleep 1
+pm2 start /rackhd.yml
+dhcpd -f -cf /etc/dhcp/dhcpd.conf -lf /var/lib/dhcp/dhcpd.leases --no-pid
```

Comment:
why do you have to run dhcpd instead of rely on isc-dhcp-server (which you stopped it in line 12)?